index.html

```diff
@@ -26,6 +26,7 @@ <h1>nav</h1>
         <span class="navbar-toggler-icon"></span>
       </button>
 
+      <!-- @matt: Should IDs use all lower-case? e.g. id="navbar-supported-content" -->
```

Comment:
This code was taken from bootstrap nav docs https://getbootstrap.com/docs/4.0/components/navbar/